tests/test_scheme.py

```diff
@@ -20,33 +34,130 @@ def test_scheme():
     assert 'label1' in db.schemes['label_map_str']
     assert 1 in db.schemes['label_map_int']
 
+    # Misc table
+    # assigned scheme
+    assert 'spk1' in db.schemes['speaker']
+    # remove table
+    # db.drop_tables(['misc'])
+    db.misc_tables.pop('misc')
```

Comment:
`db.drop_tables()` is not working for misc tables, is this on purpose?

I created https://github.com/audeering/audformat/pull/203 to change it.